src/lib/logger.ts

```diff
@@ -0,0 +1,94 @@
+export interface LogOptions {
```

Comment:
I think it would be awesome to have a `callback` option as well, where a user can pass in function that is called with every message & metadata. This would allow custom logging solutions, such as logging to kibana, a DB, etc.

Reply:
Your wish is granted